src/playbooks/deploy/metadata.obsah.yaml

```diff
@@ -47,6 +47,25 @@ variables:
     help: User of the Pulp database.
   pulp_database_password:
     help: Password for the Pulp database.
+  foreman_logging_level:
+    help: Set the log level for Foreman.
+    choices:
+      - fatal
+      - error
+      - warn
+      - info
+      - debug
+  foreman_logging_layout:
+    help: Set the structured layout type for Foreman logs.
+    choices:
+      - pattern
+      - multiline_pattern
+      - multiline_request_pattern
+      - json
+  foreman_loggers:
+    help: Enable specific loggers for debugging Foreman.
+  candlepin_loggers:
```

Comment:
Can we somehow tell obsah this is a dict? In kafo you can specify `--candlepin-loggers logger1:value1 --candlepin-loggers logger2:value2` and that results in a single hash with 2 keys.

Same above for `foreman_loggers`

Reply:
I wondered the same thing. And wanted to see what @evgeni thought here on that as a type.

Reply:
What's the end data structure you expect here?
```
candlepin_loggers:
  logger1: value1
  logger2: value2
```
That?

With "regular ansible", you'd just pass that as JSON, so `--candlepin-loggers '{"logger1":"value1","logger2":"value2"}'`. Yes, not strictly userfriendly, I know.

Kafo (judging by Ewouds comment) does allow multiple params that then get joined to one big dict?

I am sure we can implement that, just not sure it's an actual improvement for the user (I guess for the "one entry case, the syntax is nicer, but once you have more… idk)